Makefile

```diff
@@ -161,12 +161,12 @@ tf_init: ## Initialise terraform
 .PHONY: tf_plan
 tf_plan: ## Plan terraform
 	make tf_set_workspace && \
-	terraform -chdir=./infrastructure/aws plan -var-file=$(CONFIG_DIR)/${env}-input-params.tfvars ${tf_build_args}
+	terraform -chdir=./infrastructure/aws plan -var-file=$(CONFIG_DIR)/${env}-input-params.tfvars
```

Comment:
We should remove the declaration of `tf_build_args` if we're not using it

Reply:
[c31cac8](https://github.com/i-dot-ai/redbox-copilot/pull/292/commits/c31cac8682bcf0dd6e137e6a8bb48bdda0b7bdc4)